lib/experimental/Widgets/Content/EventsList/index.tsx

```diff
@@ -6,32 +7,34 @@ export interface EventsListProps {
   limit?: 1 | 2 | 3 | 4 | 5
 }
 
-export const EventsList: React.FC<EventsListProps> = ({
-  events,
-  title,
-  limit = 3,
-}) => {
-  if (!events.length) {
-    return null
-  }
+export const EventsList = forwardRef<HTMLDivElement, EventsListProps>(
+  ({ events, title, limit = 3 }, ref) => {
+    if (!events.length) {
+      return null
+    }
 
-  return (
-    <div>
-      {title && (
-        <p className="mb-4 font-medium text-f1-foreground-secondary">{title}</p>
-      )}
-      <div className="flex flex-col gap-4">
-        {events.slice(0, limit).map((item) => (
-          <Event
-            key={item.title}
-            title={item.title}
-            subtitle={item.subtitle}
-            description={item.description}
-            color={item.color}
-            isPending={item.isPending}
-          />
-        ))}
+    return (
+      <div ref={ref}>
+        {title && (
+          <p className="mb-3 text-sm font-semibold uppercase text-f1-foreground-secondary">
+            {title}
+          </p>
+        )}
+        <div className="flex flex-col gap-2">
+          {events.slice(0, limit).map((item) => (
+            <Event
+              key={item.title}
```

Comment:
We should probably use a combination of multiple properties for the key, they can have the same title:

```suggestion
              key={`${item.title} ${item.subtitle} ${item.description}`}
```

Reply:
This make sense